lte/gateway/fabfile.py

```diff
@@ -301,47 +188,6 @@ def _setup_gateway(
     return gateway_connection, gateway_ip
 
 
-@task
-def integ_test(
```

Comment:
Please migrate the relevant content from here to the docstring of `integ_test_containerized`, since its docstring refers to this function on line 242